.ci/google-java-format.sh

```diff
@@ -4,16 +4,17 @@ set -e
 
 JAR_PATH="$1"
 
-INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.java" \
+INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ \
+    \( -name "Input*.java" -o -name "InputFormatted*.java" \) \
```

Comment:
this change is not required.
` "Input*.java"` is enough and fully covering `"InputFormatted*.java"`

Reply:
Please do this

Reply:
done